config/config.go

```diff
@@ -496,6 +534,85 @@ func (c *ScrapeConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
+// AlertingConfig configures alerting and alertmanager related configs
+type AlertingConfig struct {
+	AlertRelabelConfigs  []*RelabelConfig       `yaml:"alert_relabel_configs,omitempty"`
+	AlertmanagersConfigs []*AlertmanagersConfig `yaml:"alertmanagers,omitempty"`
```

Comment:
Yea, I can just make it singular. I guess it's obvious from the context and easier to read.